examples/quickstart/run.py

```diff
@@ -74,22 +70,32 @@ def main(
 
     Args:
         model_type: Type of model to use
-        orchestration_environment: Environment where the pipeline will run
+        config_path: Configuration file to use
         no_cache: If `True` cache will be disabled.
     """
-    config_folder = os.path.join(
-        os.path.dirname(os.path.realpath(__file__)),
-        "configs",
-    )
-
+    client = Client()
     run_args_train = {}
 
+    crf = client.active_stack.container_registry.flavor
```

Comment:
The container registry can be missing from the active stack. How is mypy not even complaining here ?

Reply:
On the other hand, I would recommend not even checking the container registry here at all, because the parent container image is only relevant for infrastructure-related stack components that will have code running inside the container itself: artifact stores, orchestrators and step operators.

Reply:
good catch